recipe/meta.yaml

```diff
@@ -58,13 +60,13 @@ test:
     {% endfor %}
 
     # CLI tests
-    - genbrk --help  # [not win]
-    - gencfu --help  # [not win]
-    - gencnval --help  # [not win]
-    - gendict --help  # [not win]
-    - icuinfo --help  # [not win]
+    - genbrk --help
+    - gencfu --help
+    - gencnval --help
+    - gendict --help
+    - icuinfo --help
     - icu-config --help  # [not win]
```

Comment:
Is there no `icu-config` for Windows? Probably not a crisis if there isn't. Just curious.